packages/slice-machine/lib/builders/SliceBuilder/Header/index.tsx

```diff
@@ -55,91 +70,62 @@ const Header = ({ Model, store, variation, onSave, onPush, isLoading }) => {
                     variations={Model.variations}
                     onNewVariation={() => setShowVariationModal(true)}
-                    onChange={(v) => {
-                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-floating-promises
+                    onChange={(v) =>
                       router.push(
                         ...Links.variation({
-                          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
                           lib: Model.href,
-                          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-floating-promises, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
                           sliceName: Model.infos.sliceName,
                           variationId: v.id,
                         }).all
-                      );
-                    }}
+                      )
+                    }
                   />
                   <Box ml={2}>
-                    <Text // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                      variant="xs"
-                    >
-                      Variation id : {variation.id}
-                    </Text>
+                    <Text variant="xs">Variation id : {variation.id}</Text>
                   </Box>
                 </Flex>
               </Flex>
             </Flex>
           </Box>
 
           <SaveButton
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
             onSave={onSave}
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
             onPush={onPush}
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
             __status={Model.__status}
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
             isTouched={Model.isTouched}
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
             isLoading={isLoading}
+            imageLoading={imageLoading}
           />
           <VariationModal
             isOpen={showVariationModal}
             onClose={() => setShowVariationModal(false)}
             onSubmit={(id, name, copiedVariation) => {
-              // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
               store.copyVariation(id, name, copiedVariation);
-              // eslint-disable-next-line @typescript-eslint/no-floating-promises
-              router.push(
+              return router.push(
```

Comment:
Note: if you don't want to return it, I think you can do 
```suggestion
              void router.push(
```

Reply:
I think return works better as the process is async

Reply:
Yeah but we don't need what the function returns.
I'm supposing you put `return` for eslint.
The good way to do it for a promise on which you don't need the result is void.

But that's a `Note`, so don't change it if you don't want to. That was just me trying to spread what i consider best practices 😃